src/components/fellows-header.js

```diff
@@ -1,37 +1,12 @@
 var React = require(`react`);
+var StickyContainer  = require(`./sticky-container.js`);
 var Router = require(`react-router`);
 var Link = Router.Link;
 
 module.exports = React.createClass({
-  getInitialState: function() {
-    window.scrollTo(0, 0);
-    return {
-      sticky: false
-    };
-  },
-  onScroll: function() {
-    var sticky = window.scrollY >= 448;
-
-    if (sticky !== this.state.sticky) {
-      this.setState({
-        sticky: sticky
-      });
-    }
-  },
-  componentDidMount: function() {
-    document.addEventListener(`scroll`, this.onScroll);
-  },
-  componentWillUnmount: function() {
-    document.removeEventListener(`scroll`, this.onScroll);
-  },
   render: function() {
-    var className = `fellows-header`;
-
-    if (this.state.sticky) {
-      className += ` sticky`;
-    }
```

Comment:
Moved the above removed code into the stick container component which is now shared.

The main functionality for this is it's sticky at certain points of y scroll in the page, or fixed at other points.